qiskit_nature/second_q/algorithms/ground_state_solvers/ground_state_eigensolver.py

```diff
@@ -123,100 +111,10 @@ def get_qubit_operators(
                     )
                 aux_ops[name_aux] = converted_aux_op
 
-        if isinstance(self._solver, MinimumEigensolverFactory):
+        if isinstance(self.solver, MinimumEigensolverFactory):
             # this must be called after transformation.transform
-            self._solver = self._solver.get_solver(problem, self._qubit_converter)
+            self._solver = self.solver.get_solver(problem, self._qubit_converter)
         # if the eigensolver does not support auxiliary operators, reset them
-        if not self._solver.supports_aux_operators():
+        if not self.solver.supports_aux_operators():
             aux_ops = None
```

Comment:
This call being in here is an artifact of the the awkward `solver` property interface. It would also be useful to be able to get the `aux_ops` regardless of whether the underlying solver supports their evaluation.
In other words, the `Factory.get_solver` call should not be part of the `get_qubit_operators` method.